examples/wine-flavors.get.json

```diff
@@ -0,0 +1,95 @@
+{
+    "wine_flavors": [
+        {
+            "id": "acid",
+            "titles": [
+                {
+                    "locale": "de",
+                    "value": "Säure"
+                },
+                {
+                    "locale": "fr",
+                    "value": "..."
+                },
+                {
+                    "locale": "it",
+                    "value": "acidità"
+                }
+            ],
+            "sort_index": 1
+        },
+        {
+            "id": "sweet",
+            "titles": [
+                {
+                    "locale": "de",
+                    "value": "Zucker"
+                },
+                {
+                    "locale": "fr",
+                    "value": "..."
+                },
+                {
+                    "locale": "it",
+                    "value": "zuccheri"
+                }
+            ],
+            "sort_index": 2
+        },
+        {
+            "id": "intensity",
+            "titles": [
+                {
+                    "locale": "de",
+                    "value": "Intensität"
+                },
+                {
+                    "locale": "fr",
+                    "value": "..."
+                },
+                {
+                    "locale": "it",
+                    "value": "intensità"
+                }
+            ],
+            "sort_index": 3
+        },
+        {
+            "id": "barrique-aging",
```

Comment:
@ivan-wolf In the first version, Denner called this flavor "Veredlung" .. that would translate to "finishing"  

What do you think is the best naming for this?

Reply:
`barrique-aging` seems fine.